src/server.ts

```diff
@@ -138,8 +143,9 @@ const startServer = async () => {
     server.post('/create-hash', routeCreateHash)
     server.post('/generate-pdf', routeGeneratePDF)
     server.get('/data-views', routeDataViews)
-    server.get('/data-views/:dataViewCode', routeDataViewTable)
+    server.post('/data-views/:dataViewCode', routeDataViewTable)
```

Comment:
Had to change this one to a "POST" method, since it can now take a JSON body (containing the search filter). "GET" method can only take query parameters, which is not enough for the structured filter object.

I was previously sending query info in parameters (for pagination) and constructing it into a filter object in the back-end. However, I'm not using the same machinery used for the ApplicationList for constructing the search filter (and soon the "filtering" filter), so that has to be passed pre-made from the front-end (and is merged into the parameter-based filter here in back-end).